R/cookies.R

```diff
@@ -243,196 +279,176 @@ init_cookies <- function() {
   )
 }
 
-  #' cookies_panel_ui
-  #'
-  #' @description
-  #' Create the standard DfE R-Shiny cookies dashboard panel in the ui. The server
-  #' side functionality is provided by cookies_panel_server()
-  #'
-  #' @param id ID shared with cookies_panel_server()
-  #' @param google_analytics_key Provide the GA 10 digit key of the form
-  #' "ABCDE12345"
-  #'
-  #'
-  #' @return a standardised panel for a public R Shiny dashboard in DfE
-  #' @export
-  #'
-  #' @examples
-  #' \dontrun{
-  #' cookies_panel_ui(
-  #'   id = "cookies_panel",
-  #'   google_analytics_key = "ABCDE12345"
-  #' )
-  #' }
-  #'
-  #'
-  cookies_panel_ui <- function(
-      id, google_analytics_key = NULL) {
-    # Build the support page ----------------------------------------------------
-    shiny::tabPanel(
-      id = shiny::NS(id, "cookies_panel"),
-      value = "cookies_panel_ui",
-      "Cookies",
-      shinyGovstyle::gov_main_layout(
-        shinyGovstyle::gov_row(
-          shiny::column(
-            width = 12,
-            shiny::tags$h1("Cookies"),
-            shiny::tags$p("Cookies are small files saved on your phone, tablet or
+#' cookies_panel_ui
+#'
+#' @description
+#' Create the standard DfE R-Shiny cookies dashboard panel in the ui. The server
+#' side functionality is provided by cookies_panel_server()
+#'
+#' @param id ID shared with cookies_panel_server()
```

Comment:
Can we add a bit more explanation about the ID? Does the user decide what it's called? How do they decide what to call it?

Reply:
Added, and done this consistently across all four on the page that use an id argument